proposals/sidepanel-close-api.md

```diff
@@ -0,0 +1,132 @@
+# New close() Method for the sidePanel API
+
+**Summary**
+Proposal to introduce a programmatic close method for the side panel API.
+
+**Document Metadata**
+
+**Author:** [hharshas](https://github.com/hharshas)
+
+**Sponsoring Browser:** Chromium
+
+**Contributors:** [hharshas](https://github.com/hharshas) (
+  [Summer Of Code](https://summerofcode.withgoogle.com/)), mentor: oliverdunk, solomonkinard
```

Comment:
Placing the opening parentheses (`(`) on one line and the content inside it on the next inserts a space between the the parentheses and its content. (Visible [here](https://github.com/hharshas/webextensions/blob/255679a4b0073b51fe47cfe675d743b4f8c4da79/proposals/sidepanel-close-api.md))

```suggestion
**Contributors:** [hharshas](https://github.com/hharshas) ([Summer Of
Code](https://summerofcode.withgoogle.com/)), mentor: oliverdunk, solomonkinard
```